src/features/ResetPassword/ResetPasswordPage.jsx

```diff
@@ -35,7 +35,12 @@ const ResetPasswordPage = (props) => {
       },
       onError: (error) => {
         if (error.response.data && error.response.data.message) {
-          const errorMessage = error.response.data.message;
+          let errorMessage = error.response.data.message;
+          if (errorMessage === "user not found: record not found") {
+            errorMessage =
+              "Something went wrong. Please check if you entered a valid email";
```

Comment:
Can we use this message instead? Also lets ensure that this message change happens on server.

If an account exists with that email, you'll receive a password reset link. 

Use the same message for both success and failure.